reference/customers.v3.yml

```diff
@@ -712,6 +713,62 @@ paths:
           headers: {}
       tags:
         - Customer Addresses
+  /customers/validate-credentials:
+    post:
+      tags:
+        - Customer Validate Credentials
+      description: Validate a customer credentials - This endpoint has special rate limiting protections to protect against abuse
+      summary: Validate a customer credentials
+      operationId: CustomerValidateCredentials
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ValidateCustomerCredentialsObject'
+            examples:
+              data:
+                value:
+                  email: 'foo@bar.com'
+                  password: 'password'
+                  channel_id: 1
+      responses:
```

Comment:
Missing `422` response, which gets returned if the request parameters are incorrect?